src/distributors/zulu-installer.ts

```diff
@@ -77,6 +77,11 @@ export class ZuluDistributor extends JavaBase {
     const platform = this.getPlatformOption();
     const extension = getDownloadArchiveExtension();
     const javafx = features?.includes('fx') ?? false;
+    let release_status = 'ga';
```

Comment:
nitpick: release_type -> releaseType
nitpick2: const releaseType = this.stable ? 'ga' : 'ea'

Reply:
Done. Thank you.